Week-2/Session-6/Recipe/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>binary eats - Oatmeal Coconut Pantry Cookies</title>
+      <link href="styles.css" rel="stylesheet">
+   </head>
+
+   <body>
+      <header>
+         <div class="logo">binary eats</div>
+         <div>
+            <a href="">sign-in</a>
+            <a href="">search</a>
+         </div>
+      </header>
+      <div class="content">
+         <div class="main-content">
+            <div class="title">
+               <h1>OATMEAL COCONUT PANTRY COOKIES</h1>
+               <p>Chef Roman, August 24, 2018</p>
+            </div>
+            <div class="summary">
+               <img src="oatmealCookie_small.jpg" alt="Photo of two of the world's best oatmeal cookies">
+               <div>
+                  <div>
```

Comment:
We can use a more semantic approach here and instead of using divs, we can also use sections and articles HTML tag to make it more semantic.

Reply:
hey @kamalmohit 
This was part of the boilerplate code provided hence didn't modify it